node-notes-cli/update.js

```diff
@@ -0,0 +1,12 @@
+const json = require('./data.json');
+const fs = require('fs');
+
+module.exports = (a, b) => {
```

Comment:
`a` and `b` aren't very descriptive names! Why do I have to read the code (and know what it's doing) to figure out what they are used for? :-)